db-service/lib/cqn2sql.js

```diff
@@ -795,25 +792,7 @@ class CQN2SQLRenderer {
   }
 
   /**
-   * Renders a Regular Expression into its string representation
```

Comment:
This is currently being called by the HANA service in the tests. Should simply be adjusting the `val` overwrite in the HANA Service.

Reply:
Let's revisit in that context with HANA currently copying the whole impl of `val()`